share-credential-modal.html

```diff
@@ -0,0 +1,10 @@
+<br-modal br-title="Share Credential">
+  <div name="br-modal-body">
+    <input type="text" value="{{shareUrl}}" br-select-text="true" size="50"></input><br>
+    Press <strong>Ctrl + C</strong> to copy URL
```

Comment:
Do we want to strike the copy instructions entirely, or add instructions for the Mac as well?